### DISCUSSION TITLE
Request: Stream-Only Option with no Motion Detection?

### DISCUSSION THREAD

**Discussion by tokenwizard** [Motion-Project/motion#1517] [General]
0. Reviewed guide and contributing documents? (Yes/No): Yes
1. version [x.y.z, hash, other]: 4.2.2
2. installed as a package or compiled from sources [deb, rpm, git, other]:  Installed via Ubuntu Repositories (apt-get)
3. standalone or part of third party [motion, MotionEyeOS, other]: Standalone
4. video stream source [V4L (card or USB), net cam (mjpeg, rtsp, other), mmal]: RTSP
5. hardware [x86, ARM, other]: KVM - X86_64
6. operating system [16.04, Stretch, etc, FreeBSD, other]: Ubuntu Server

Not really an Issue as much as a question/suggestion. 

Goal: Have a single, simple HTML page to display live streams from multiple RTSP cameras

I don't actually need Motion to perform motion detection or any other processing other than re-streaming in an HTML/HTTP-friendly format. 

I have looked through all the available parameters in the Motion documentation, and I have disabled the video and picture saving options, but there does not seem to be a way to disable motion detection. 

Am I missing it, or is this something that could be added? I want to mount Monitors in several locations throughout the house with Raspberry Pis running Firefox in Kiosk Mode to display these streams in a grid view. 

Thanks!

**Comment by Mr-Dave** [comment]
[pause](https://motion-project.github.io/motion_config.html#pause)?

**Comment by tokenwizard** [comment]
So this will only pause motion detection but still allow streaming the live feed? If so, that should do the trick. Thanks!

**Comment by tokenwizard** [comment]
I'm running 4.2.2 and added "pause on" to the motion.conf file but it throws this error when starting motion:
![image](https://user-images.githubusercontent.com/24832561/160467013-a832e0c7-9a34-4c58-8f70-d3bbb2859e40.png)

I had mentioned version 4.2.2 in my original post. It looks like I will have to manually install from source to get to a current version on Ubuntu so that I can use this pause option.

**Comment by tokenwizard** [comment]
Just to close the loop, I installed the latest release deb in Ubuntu 20 LTS and was able to use the pause function to do what I needed. 

Thanks!

**Comment by scy** [comment]
Let me just ask a follow-up here:

Using the `pause` option should do basically the same as clicking the “Pause” action in the web control, right? However, at least with 4.3.2 I don’t see any change in CPU usage between a paused Motion and an unpaused one.

Is the actual motion detection effort too small to notice? Like, if I have Motion running with 4 cameras at 4 fps, creating stream JPEGs, a time lapse and motion detection with `despeckle_filter EedDl`, using 35 % CPU, how much of that can I expect to be the actual motion detection?

**Reply by PhatHub** [reply]
What hardware are you running? 

I'm curious too, because I'd like to run an external esp8266 PIR/Radar tripwire to trigger the camera recording, because my Core2Duo setup with 4gb RAM is choking on five cameras (three HD@h265, two 800x600@mjpeg) . I'm assuming that the choke is due to digesting frames for motion detection. This causing my fps to go from 15 (with four cameras) to somewhere between 3fps to >1sec per frame. I had to deactivate two of the cameras.

I'd figure the `detection/start` and `detection/pause` portion of the [API](https://motion-project.github.io/motion_config.html#OptDetail_Webcontrol) would be my target as I could probably create a service that subs an MQTT feed to send an http GET to the URLs when my esp8266s pubs to the server... but if it's still going to peg the processor while "paused," I might not make the effort to create that service.

**Comment by tokenwizard** [comment]
I've long since shifted from using Motion. I spent some time with several options like Zoneminder.

Now I'm using a system I'm helping the developer test that is based on Go and is pretty lightweight.

These days I'm running a Proxmox LXC with my NVR running in Docker.

Sent from Proton Mail mobile

-------- Original Message --------

**Comment by Mr-Dave** [comment]
Just a FYI.  By far the largest use of the CPU is decoding the image.  I would not expect that turning off the Motion detection would have a material effect.  On the other hand changing from H265 to MJPG would drop CPU usage dramatically (while of course increasing bandwidth)

**Comment by PhatHub** [comment]
Uh, just to update on my (tangential) part, I tested manually turning on pause with all my streams, and it still pegged my Core2Duo and I also noticed that my streams were getting motion triggered (it started snowing) so I'm not sure what's up with that. 
I also tried to activate `emulate_motion` while motion detection setting was off, and that didn't trigger a recording. I guess I'll shelve my hard tripwire/MQTT server idea. :disappointed:

**Comment by onyxiarivera** [comment]
You can add option `threshold 0` to your config file to completely disable motion detection on process start without a curl request :)
